### DISCUSSION TITLE
One to many polymorphic relationship

### DISCUSSION THREAD

**Discussion by ShaungBhone** [filamentphp/filament#12024] [Help]
### Package

Table builder

### Package Version

v3

### How can we help you?

How can I retrieve each model in appointment table to filament table column?
Example
```php
$type_name = appointment()->type->name;
$make_name = appointment()->make->name;
```
![Screenshot_from_2024-03-23_23-30-08 (1)](https://github.com/filamentphp/filament/assets/56265471/eb9d61f2-0457-447a-a3ea-9954bb235b16)

**Comment by wertw3t** [comment]
$name = appointment()->type->name ?? appointment()->make->name;
